cmd/libsnap-confine-private/mountinfo.c

```diff
@@ -241,7 +283,6 @@ static sc_mountinfo_entry *sc_parse_mountinfo_entry(const char *line)
 	if ((entry->super_opts =
 	     parse_next_string_field(entry, line, &offset)) == NULL)
 		goto fail;
-	show_buffers(line, offset, entry);
```

Comment:
Was this a debug output?

Reply:
Yes, but it was duplicated because `parse_next_string_field` also prints it so I just decided to remove this one.